src/Common/tests/System/Collections/ICollectionTest.cs

```diff
@@ -125,6 +125,9 @@ public void CopyToBadIndex(int index)
         [Fact]
         public void CopyToArrayWithNonZeroBounds()
         {
+            if (!PlatformDetection.IsNonZeroLowerBoundArraySupported)
```

Comment:
For cases like this where we want to skip the whole test based on some condition, we should use [ConditionalFact] or [ConditionalTheory].